docs/customizing-evaluation-outputs.md

```diff
@@ -0,0 +1,54 @@
+# Template BPA Customizing Evaluation Outputs
+<a name="note"></a>***Note**: The ARM Template BPA is currently in development. All features that have yet to be implemented have been flagged with an asterisk [\*].*
+
+## Overview
+Template BPA customization is authored in JSON.  Custom evaluation settings are written in a separate configuration and used on tool executing. More details on how to run Template BPA with a configuration file can be found [here](https://github.com/Azure/template-analyzer#using-the-template-bpa). 
+
+## Template BPA Rule Object
+Here are the fields that make up a custom configuration file.
+```javascript
+{
+    "exclusions": { � }, // Parameters that are excluded from the BPA execution.  More details below.
```

Comment:
This work would also close #132 and #133, it probably makes sense to rename the PR

Reply:
I think we need to write "Closes #issue" in its own line for each so that the issues get closed when the PR is merged

Reply:
Right - when GitHub detects the "closing language", the issue will show up on the right side bar under "Linked issues"

Reply:
fixed, thanks!